### DISCUSSION TITLE
Blazor wasm

### DISCUSSION THREAD

**Discussion by mirmajeedJet** [OrchardCMS/OrchardCore#14567] [Q&A]
Where do we currently stand with blazor wasm support? Is there any guidance or documentation on how to integrate Blazor wasm with Orchard Core?

**Comment by ns8482e** [comment]
Blazor wasm work well with Orchard Core  see sample https://github.com/surevelox/OrchardCore.Samples/tree/ns8482e/blazor-wasm/MultiTenantBlazorWasm
